src/components/navbar/_navbar.scss

```diff
@@ -5,6 +5,7 @@
     width: 100%;
     top: 0;
     left: 0;
+    z-index: 10001;
```

Comment:
`z-index` should have constants for each related components. `10001` is not good prefer :)